src/cmake/GeosxOptions.cmake

```diff
@@ -93,6 +93,16 @@ endif(NOT BLT_CXX_STD STREQUAL c++14)
 
 message("CMAKE_CXX_COMPILER_ID = ${CMAKE_CXX_COMPILER_ID}")
 
+option( ENABLE_JITTI "Build all compute kernels just-in-time at runtime." OFF )
+
+if ( ENABLE_JITTI )
+  message( "JITTI is ENABLED")
```

Comment:
I think the way we've been doing this is `GEOSX_ENABLE_JITTI`, or maybe it's `GEOSX_USE_JITTI`, I forget. But either way I like defined/not defined instead of 1/0.